tests/core/test_project.py

```diff
@@ -370,6 +370,36 @@ def test_from_file(speos: Speos):
     assert ssr_data.irradiance_sensor_template.dimensions.x_sampling == 500
 
 
+def test_from_file_threads_limited(speos: Speos):
+    """Test create a project from file."""
+    # Create a project from a file
+    p = Project(
+        speos=speos,
+        path=str(
+            Path(test_path) / "LG_50M_Colorimetric_short.sv5" / "LG_50M_Colorimetric_short.sv5"
+        ),
+    )
+
+    # Check that scene is filled
+    assert len(p.scene_link.get().materials) == 4
+    assert len(p.scene_link.get().sensors) == 1
+    assert len(p.scene_link.get().sources) == 2
+    assert len(p.scene_link.get().simulations) == 1
+
+    feat_sims = p.find(name=p.scene_link.get().simulations[0].name)
+    assert len(feat_sims) == 1
+    assert type(feat_sims[0]) is SimulationDirect
```

Comment:
```suggestion
    feat_sims = p.find(name=p.scene_link.get().simulations[0].name)
```
I think we already tested that on file loading do we need to do it twice?